code/modules/projectiles/guns/rifles.dm

```diff
@@ -109,8 +117,14 @@
 	name = "\improper AR-12 K&H assault rifle"
 	desc = "The Keckler and Hoch AR-12 assault rifle used to be the TerraGov Marine Corps standard issue rifle before the AR-18 carbine replaced it. It is, however, still used widely despite that. The gun itself is very good at being used in most situations however it suffers in engagements at close quarters and is relatively hard to shoulder than some others. It uses 10x24mm caseless ammunition."
 	icon = 'icons/marine/gun64.dmi'
-	icon_state = "t12"
+	icon_state = "loaded"
```

Comment:
we dont put all the other icon_state stuff in defines, why should we for this? Besides, this default is only for making the guns appear correctly on the loadout UI

Reply:
Because this is a default shared among multiple guns. It has to be clearly stated that is it greyscaled and a define is much more clearer than a string.